app/server/lib/FlexServer.ts

```diff
@@ -1132,6 +1131,7 @@ export class FlexServer implements GristServer {
         this._disableExternalStorage = true;
         externalStorage.flag('active').set(false);
       }
+      await this.create.configure?.();
```

Comment:
That seems reasonable to me.

Reply:
Thanks for your feedback!

Reply:
I was wrong about this, your `unless the checks change the settings read above?` qualm was justified. Sorry for not looking more carefully. A staging deployment we have broke because of this change. Perhaps it would be better to separate out the checkBackend phase from the configuration step @fflorent?